### DISCUSSION TITLE
Ventilstatus bei Heizungsthermostat HM-IP HmIP-eTRV / HmIP-eTRV-2

### DISCUSSION THREAD

**Discussion by KaJu99** [Zefau/ioBroker.jarvis#1616] [Q&A]
**Is your feature request related to a problem? Please describe.**
Jarvis Version 3.0.11

der Ventilstatus der HM-IP Geräte wird unter valve mit Prozentangabe angezeigt (bei mir wird statisch 4% angezeigt); die Anzeige greift dabei auf den "Valve Status" zurück, der blöderweise gar nicht die Ventilöffnung liefert, sondern m. W. irgendein Ergebnis aus der Adptierfahrt. Ob das nun ein Bug oder ein fehlendes Feature seitens Homematic ist sei dahingestellt. Fakt ist, dass HM leider gar keinen prozentuale Öffnungswert bereitstellt; insoweit ist die Anzeige in Jarvis irreführend. In Foren findet man daher Ideen, das Problem mit Variablen in der CCU zu lösen, habe ich jetzt aber nicht ausprobiert.

----- OR -----

**Describe the solution you'd like**
A clear and concise description of what you want to be added.

**Comment by Zefau** [comment]
Das heißt? Soll ich es ganz raus nehmen oder was wäre eine Lösung?

**Comment by KaJu99** [comment]
ich würde es drin lassen, ggf. passt das eQ3 ja mal an; die Information liegt ja grundsätzlich vor und bei den "normalen", nich HM-IP Thermostaten funktioniert das ja auch. Ist also eher als Info zu verstehen, ich wusste nicht, wo ich das sonst los werde.
